src/global/colors.js

```diff
@@ -35,19 +35,20 @@ export const isEmpty = v => !(typeof (v) !== 'undefined' && v);
 // example: convertHex("#FF120AE",0.3)
 // example: convertHex("#FF120AE",30)
 export const convertHex = (hex, opacity) => {
-  hex = hex.replace('#', '');
-  const r = parseInt(hex.substring(0, 2), 16);
-  const g = parseInt(hex.substring(2, 4), 16);
-  const b = parseInt(hex.substring(4, 6), 16);
+  const HEX = hex.replace('#', '');
```

Comment:
I see that ESLint force us to not use assignment on parameters, however edited version seems to be more messy and unclear than original. I guess naming change could help, for example, call it adjustedHex or something to make it clear by naming what is the difference between hex and HEX.